src/extension/package.json

```diff
@@ -53,10 +53,10 @@
       "type": "object",
       "title": "%webTemplateStudioExtension.commands.wts%",
       "properties": {
-        "wts.defaultOutputPath": {
+        "wts.changeSaveToLocation": {
           "type": "string",
           "default": "",
-          "description": "Load a default output path on wizard launch."
+          "description": "Change your save to location for your project."
```

Comment:
Confirmed this work on windows :)

We discussed this string with the team and we think `Set default project location for your project` would be more clear because the word `change` might be confusing. What do you think @jasmingoh